### PR TITLE
[CircleEditor] Add Input and Output Edit Function

### PR DESCRIPTION
**Pull request by rudalsd**
This PR adds input and output edit function.
It is possible to change tensor name, type and shape.

ONE-vscode-DCO-1.0-Signed-off-by: Gyeongmin Kwon rudalsd@naver.com

### CODE REVIEW

File: media/CircleEditor/view-sidebar.js [hunk 1]
```diff
@@ -993,9 +1016,24 @@ sidebar.ParameterView = class {
...
+        this._data;
+        this._index = ioAttributes.index;
+        this._isCustom = ioAttributes.isCustom;
+        this._tensors = tensors;
+        this._title = ioAttributes.title;
+
+        this._editObject = {
+            _name: ioAttributes.name,
+            _visible: ioAttributes.visible,
+            _arguments: JSON.parse(JSON.stringify(argument)),
```
Comment (llFreetimell) [inline media/CircleEditor/view-sidebar.js:1033]: Isn't it `JSON.parse(JSON.stringify(argument)) === argument` ?

Comment (rudalsd) [reply media/CircleEditor/view-sidebar.js:1033]: If do not use this method when sending an object value, the value itself may change even if the value is not valid.  So we have to deep copy.

File: media/CircleEditor/view-sidebar.js [hunk 2]
```diff
@@ -1141,28 +1142,319 @@ sidebar.ArgumentView = class {
...
+        }
+
+        while (this._element.childElementCount) {
+            this._element.removeChild(this._element.lastChild);
+        }
+        const initializer = this._argument.initializer;
+        this.show(initializer);
+    }
+
+    check() {
```
[the remark this answers is not in the document]
Comment (rudalsd) [reply media/CircleEditor/view-sidebar.js:1450]: The check function simply checks whether there are only numbers and commas, and 1,2,,,,3 occurs error.

**Review by llFreetimell — APPROVED** [review]
LGTM

**Review by YuyeonHan — APPROVED** [review]
LGTM :)

**Pull request merged**